closed/autoconf/custom-hook.m4

```diff
@@ -110,7 +110,14 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
   OPENJ9_BUILDSPEC="${OPENJDK_BUILD_OS}_${OPENJ9_CPU}_cmprssptrs"
 
   if test "x$OPENJ9_CPU" = xx86-64; then
-    OPENJ9_PLATFORM_CODE=xa64
+    if test "x$OPENJDK_BUILD_OS" = "xlinux"; then
+      OPENJ9_PLATFORM_CODE=xa64
+    elif test "x$OPENJDK_BUILD_OS" = "xwindows"; then
+      OPENJ9_PLATFORM_CODE=wa64
+      OPENJ9_BUILDSPEC="win_${OPENJ9_CPU}_cmprssptrs"
```

Comment:
Why the reference to `OPENJ9_CPU` when we know it is `x86-64`?
The double-quotes are not necessary in `"xlinux"` and `"xwindows"`: please remove them for consistency.